Singularity

```diff
@@ -0,0 +1,42 @@
+Bootstrap: docker
+From: ubuntu:devel
+# Requires: Singularity 2.3+
+
+%help
+Welcome to the openPMD-api container.
+This container contains a pre-installed openPMD-api library.
+It supports serial and MPI parallel I/O with HDF5 and ADIOS1 and provides python bindings.
+
+%setup
+    mkdir -p ${SINGULARITY_ROOTFS}/opt/openpmd-api
+
+%files
+    ./* /opt/openpmd-api
+
+%post
+apt-get update && \
+apt-get install -y --no-install-recommends \
+    cmake \
+    ipython3 \
+    pybind11-dev \
+    libhdf5-openmpi-dev \
+    libopenmpi-dev && \
+rm -rf /var/lib/apt/lists/*
+# libadios-openmpi-dev
+
+cd $(mktemp -d)
+cmake /opt/openpmd-api \
+    -DopenPMD_USE_MPI=ON \
+    -DopenPMD_USE_HDF5=ON \
+    -DopenPMD_USE_ADIOS1=OFF \
+    -DopenPMD_USE_ADIOS2=OFF \
+    -DopenPMD_USE_PYTHON=ON \
+    -DPYTHON_EXECUTABLE=$(which python)
+make
+make install
+
+#%test
```

Comment:
for later: probably have to change dirs again and need to keep the build dir for that... maybe we don't want that.